tensorflow/c/c_api.cc

```diff
@@ -645,12 +645,12 @@ TF_DEVICELIST_METHOD(uint64_t, TF_DeviceListIncarnation, incarnation(), 0);
 
 // Helper functions -----------------------------------------------------------
 
-namespace {
-
 TF_Operation* ToOperation(Node* node) {
```

Comment:
I don't understand how this can be used in the public C API since tensorflow::Node is not a C type?

Reply:
Hmm, yeah, it's not really part of the C API, but we (tensorflow/java) are binding parts of the C++ API as well, and need this to go between.  It probably shouldn't be exposed here, but it would be nice for it to be exposed somewhere, is there a better place to put it?

Reply:
I don't think we expose any APIs that mix C and C++ symbols. Would it be possible for tensorflow/java to fork this since this is a simple cast?

Reply:
@saudet advice?  Is this something we can do in an included file like adapters?

Reply:
This is just a reinterpret cast? Just taking its address with `new TF_Operation(node)` should work: https://github.com/tensorflow/java/blob/master/tensorflow-core/tensorflow-core-api/src/gen/java/org/tensorflow/internal/c_api/TF_Operation.java#L19